.goreleaser.yml

```diff
@@ -220,6 +222,12 @@ nfpms:
           mode: 0644
           owner: root
           group: root
+      - src: dist/updater_{{ .Os }}_{{ if eq .Arch "amd64" }}amd64_v1{{ else if eq .Arch "arm64" }}arm64_v8.0{{ else if eq .Arch "arm" }}arm_{{ .Arm }}{{ else if eq .Arch "ppc64le" }}ppc64le_power8{{ else if eq .Arch "ppc64" }}ppc64_power8{{ end }}/updater
+        dst: /opt/observiq-otel-collector/updater
+        file_info:
+          mode: 0750
+          owner: observiq-otel-collector
+          group: observiq-otel-collector
```

Comment:
Conditionally copies the updater binary from the build output directory (`dist/`) into the package at `/opt/observiq-otel-collector/updater`.

This allows the `bindir` to be `/usr/bin` for the collector binary while preserving the `updater` binary's location.